TestApp/README.md

```diff
@@ -0,0 +1,41 @@
+# Basic Video Chat sample
+
+This sample application shows how to connect to an OpenTok session, publish a stream, and subscribe to multiple streams for both iOS and Android using the OpenTok React Native SDK.
+
+## Setup
+
+1. Install the required node modules: `npm install`
```

Comment:
Will the same instructions of sample work for tis repo as well?

Reply:
Yes